README.md

```diff
@@ -14,19 +14,12 @@ As of version 2.0, the lowest Supported version is 4.6.1
 ### Creating (Encoding) Tokens
 
 ```csharp
-var payload = new Dictionary<string, object>
-{
-    { "claim1", 0 },
-    { "claim2", "claim2-value" }
-};
-var secret = "GQDstcKsx0NHjPOuXOYg5MbeJ1XT0uFiwDVvVBrk";
-
-IJwtAlgorithm algorithm = new HMACSHA256Algorithm();
-IJsonSerializer serializer = new JsonNetSerializer();
-IBase64UrlEncoder urlEncoder = new JwtBase64UrlEncoder();
-IJwtEncoder encoder = new JwtEncoder(algorithm, serializer, urlEncoder);
+var token = new JWTBuilder.Builder().

```

Comment:
I still believe it should be just `new JwtBuilder()`. To avoid chattiness and tautology, e.g. Builder.Builder